.ci/build-kit/build_and_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -e
+
+cmake \
+    -B build \
+    -S "$EXT_MOUNT/source" \
+    -G Ninja \
+    -DISO15118_BUILD_TESTS=ON \
```

Comment:
As already mentioned this flag should be changed somewhen according https://everest.github.io/nightly/general/faq.html#testing

Reply:
Yes, I will change this when I adjust the CMakeLists.txt